pkg/cmd/config.go

```diff
@@ -90,6 +96,20 @@ func setConfigValue(key, value string) error {
 	return saveConfig()
 }
 
+func listAllConfig(out io.Writer) error {
+	config, err := unmarshalConfig()
+	if err != nil {
+		return nil
+	}
+
+	// Iterating over structs is hard and there are only two configuration
+	// directives at time of writing. Hard-code the output format for now, and
+	// invest in a more maintainable method in the future.
+	_, err = fmt.Fprintf(out, "user.name=%s\nuser.email=%s\n",
+		config.UserName, config.UserEmail)
+	return err
```

Comment:
Thanks Mark, that is an interesting suggestion. My immediate thoughts is that there is a requirement to add a machine-readable output format to all the commands (a `json` output), which will mean adding an output layer on top of all current output - that should be the place to format the stuct.

The branch as it stands today would probably benefit from your suggestion but I probably won't move it up now.